modules/virt-importing-vm-cli.adoc

```diff
@@ -98,37 +98,40 @@ spec:
   source:
     ovirt:
       vm:
-        id: <80554327-0569-496b-bdeb-fcbbf52b827b> <3>
-      mappings: <4>
+        id: <source-vm-id> <3>
+        name: <source-vm-name> <4>
+      cluster:
+        name: <source-cluster-name> <4>
```

Comment:
I think I understand the idea behind this, and appreciate the minimalism, but <source-vm-name> and <source-cluster-name> should have separate callouts since they do not have the same value. In the build doc it looks like a mistake and I can see confusion stemming from this. 
And callouts should be used to describe the value prior to any additional information. For example:
<3> The UUID for the source virtual machine. If you specify the source VM ID then...

Reply:
Sentence fragments are OK? Fine.